app/controllers/api/v1/ecf_participants_controller.rb

```diff
@@ -42,8 +42,9 @@ def lead_provider
 
       def participants
         participant_profiles = ParticipantProfile::ECF.where(id: participant_profile_ids)
-                                                      .joins(:user)
+                                                      .joins(:participant_identity, :user)
```

Comment:
Will this change not break the API response before we backfill the data?

Reply:
Yes, good catch. Will need to pull some of this out and add in after backfilling ...